src/promptflow/promptflow/batch/_batch_engine.py

```diff
@@ -392,8 +404,17 @@ async def _exec_batch(
     ) -> List[LineResult]:
         worker_count = self._worker_count or DEFAULT_CONCURRENCY
         semaphore = asyncio.Semaphore(worker_count)
+
+        chat_group_orchestrator = None
+        if self._chat_group_roles is not None:
+            chat_group_orchestrator = ChatGroupOrchestrator(
+                self._chat_group_roles,
+                run_id,
+                self._max_turn
+            )
```

Comment:
Why need this part?

Reply:
Per offline sync, initialize orchestrator in chat group proxy